tembo-cli/src/cmd/delete.rs

```diff
@@ -32,7 +30,7 @@ pub fn execute() -> Result<(), anyhow::Error> {
 }
 
 fn execute_tembo_cloud(env: Environment) -> Result<(), anyhow::Error> {
-    let instance_settings: HashMap<String, InstanceSettings> = get_instance_settings()?;
+    let instance_settings = get_instance_settings(None)?;
```

Comment:
Since you can't overlay `instance_name` I think passing None here should be fine.